src/backend/gpopt/translate/CContextQueryToDXL.cpp

```diff
@@ -12,12 +12,10 @@
 //		and the caller is responsible for freeing it
 //
 //---------------------------------------------------------------------------
-extern "C" {
-#include "postgres.h"
-}
-
 #include "gpopt/translate/CContextQueryToDXL.h"
+
```

Comment:
Why has this empty line been added?

Reply:
To get rid of this `clang-format` warning:
```
clang-format --dry-run src/backend/gpopt/translate/CContextQueryToDXL.cpp
src/backend/gpopt/translate/CContextQueryToDXL.cpp:15:1: warning: code should be clang-formatted [-Wclang-format-violations]
#include "gpopt/translate/CContextQueryToDXL.h"
```